sys/shell/commands/sc_netif.c

```diff
@@ -76,14 +77,21 @@ static void _set_usage(char *cmd_name)
     printf("usage: %s <if_id> set <key> <value>\n", cmd_name);
     puts("      Sets an hardware specific specific value\n"
          "      <key> may be one of the following\n"
+#ifdef MODULE_BLE_LL
+         "       * \"access_addr\" - sets the BLE access address\n"
+         "       * \"adv_addr\" - sets the BLE advertising address\n"
+#else
          "       * \"addr\" - sets (short) address\n"
          "       * \"addr_long\" - sets long address\n"
          "       * \"addr_short\" - alias for \"addr\"\n"
+#endif
          "       * \"channel\" - sets the frequency channel\n"
-         "       * \"chan\" - alias for \"channel\""
+         "       * \"chan\" - alias for \"channel\"\n"
+#ifndef MODULE_BLE_LL
```

Comment:
The channel is common to both ble and non ble so I don't think that would work. I can't think of a way to make it less confusing though, without reordering things.